src/test/app_test.js

```diff
@@ -61,7 +61,31 @@ describe("ArticleList", function() {
     var view = TestUtils.renderIntoDocument(
       <ArticleList articles={[]} />);
 
-    expect($$(view, ".list-empty")).not.eql(null);
+    expect($(view, ".list-empty")).not.eql(null);
+  });
+
+  it("should render an import button when the list is empty", function() {
+    var view = TestUtils.renderIntoDocument(
+      <ArticleList articles={[]} />);
+
+    expect($(view, ".list-empty button")).not.eql(null);
+  });
+
+  it("should not render an import button when the list is non-empty", function() {
```

Comment:
Yes, with samples you're right... Let's keep that for later when importing local files for instance :)